src/platform/stm32f1/systick.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier: MIT
+ * SPDX-FileCopyrightText: 2021 Rafael Silva <silvagracarafael@gmail.com>
+ */
+
+#pragma once
+
+#include "util/types.h"
+
+extern volatile u64 systemTick;
```

Comment:
how do you want to handle this? want a function to recall a pointer to this?

Reply:
Can it be updated outside `systick.c`? If yes let's export it here, if not, we add a function returning the value (not the pointer!).

Also, please call it `system_tick`.

Reply:
`system_tick` is only present on the header, it's typo.

I don't know why i was thinking a pointer was needed, a function exporting the value is a rather obvious answer. (this is only incremented here, in the ISR)